### DISCUSSION TITLE
BNT 2

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#482] [Show and tell]
### Channel ID

BNT2.bg

### Stream URL (optional)

https://cdn3.gledam.xyz/hls/bnt-2/index.m3u8?e=1692277965&hash=J8jiicpd2i4MfWKcNwkJJg

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://cdn3.gledam.xyz/hls/bnt-2/index.m3u8?e=1692277965&hash=J8jiicpd2i4MfWKcNwkJJg (offline)
